src/org/rascalmpl/repl/rascal/RascalInterpreterREPL.java

```diff
@@ -132,14 +142,40 @@ public IDEServices initialize(Reader input, PrintWriter stdout, PrintWriter stde
             throw new IllegalStateException("Already initialized");
         }
         eval = buildEvaluator(input, stdout, stderr, services);
+
+        // Register watches for all watchable locations on the search path for automatic reloading
+        eval.getRascalResolver().collect().stream().filter(this::isWatchable).forEach(p -> {
+            try {
+                reg.watch(p, true, d -> sourceLocationChanged(p, d));
+            }
+            catch (IOException e) {
+                e.printStackTrace(stderr);
```

Comment:
we should print a small message above this? like "Failed to watch <p> for changes to the modules" ? Or something similar, to avoid just printing an exception.